src/Storage/Doctrine/Connection.php

```diff
@@ -167,12 +174,16 @@ private function executeQuery(string $sql, array $parameters = [], array $types
 
     private function setup(): void
```

Comment:
This method is still called above from `executeQuery()` if the the table isn't found. I'm thinking that we either:

A) Should make this configurable and default it to false
B) or, simpler, just NEVER do this and remove it completely. The idea would be that we provide the listener so that you can generate the migration (or run `doctrine:schema:update`) and so THAT is what you should do, not rely on this automatic setup.

Reply:
If I look at what is done for messenger or cache adapter, I can see that schema can be updated in two ways:

### Cache
1. with the schema listener
2. or after a `TableNotFoundException`, the same way I did it in `src/Storage/Doctrine/Connection.php` https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Cache/Adapter/DoctrineDbalAdapter.php#L275-L282

### Messenger
1. with the schema listener
2. in `Doctrine\Connection::setup()` https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php#L247

I was thinking it would be better to keep it in order to prevent any error...

But I'm okay to remove the automatic stuff, I'll add an explicit error. Maybe we could also add a warning about the needed migration to generate in the readme (and maybe also the bundle's recipe? I don't know if we can add some text after `composer req` thanks to the recipes)

Reply:
Excellent! Good plan about the readme and recipe (yes, we can still add post install text)